ilc/client/ClientRouter.js

```diff
@@ -76,7 +76,10 @@ export default class ClientRouter extends EventEmitter {
     navigateToUrl = (url) => this.#singleSpa.navigateToUrl(this.#i18n.localizeUrl(url));
 
     getRelevantAppKind(appName, slotName) {
-        const appKind = this.#registryConf.apps[appName].kind;
```

Comment:
what if there is no app registered with `appName`?

Reply:
In that case verification will be invoked earlier and app will be failed with 500 error with explanation which app is not exist.

Reply:
Since method is public there is no guarantee that someone will not call it without validation, so I would rather accept `appKind` as method parameter or add validation here as well